### ISSUE TITLE
Mask only the beginning of email address

### ISSUE THREAD

**Issue by RyanEwen** [Sumukha1496/maskdata#8]
Is there a way to only mask start characters without masking anything after the @ symbol?

I have tried this but it results in the entire email address being unmasked:

```
MaskData.maskEmail('some.name@gmail.com`, {
    unmaskedStartCharacters: 2,
    unmaskedEndCharacters: 99,
    maxMaskedCharactersAfterAtTheRate: 0 // this line makes no difference
})
```

**Comment by RyanEwen** [comment]
I think the issue here is that `unmaskedStartCharacters` and `unmaskedEndCharacters` do not stop at the @ symbol. There's no way to limit masking to one side or the other.

**Comment by Sumukha1496** [comment]
@RyanEwen , Thank you for raising this issue. I am working on it. I will come up with the fix soon.

**Comment by Sumukha1496** [comment]
@RyanEwen , Fixed this issue in v1.1.0.
Please go through the Release Features mentioned in the documentation(README.md).

You can use maskEmail2. I have added this specific example in the documentation. Hope this helps. Let me know if you face any issues.

**Comment by RyanEwen** [comment]
This is great - thank you!
